src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx

```diff
@@ -44,44 +47,50 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
       {showFooterAndCorrectNumber.map((table: TableFinances, index) => (
         <TableContainer isLight={isLight} className={className} key={index} hasOthers={table.others || false}>
           <TableBody isLight={isLight}>
-            {table.rows.map((row: ItemRow, index) => (
-              <TableRow key={index} isLight={isLight} isMain={row.isMain}>
-                <Headed isLight={isLight} period={period}>
-                  {row.name}
-                </Headed>
+            {table.rows.map((row: ItemRow, index) => {
+              console.log('row.codePath', row.codePath?.replace('atlas', ''));
```

Comment:
did you forget to remove this `console.log`?

Reply:
done